### ISSUE TITLE
FP unit-no-unknown when unit is followed by back-tick

### ISSUE THREAD

**Issue by plumbblake** [stylelint/stylelint#4768]
> Clearly describe the bug

There are false positives when unit is followed by a back-tick.

> Which rule, if any, is the bug related to?

`unit-no-unknown `

> What code is needed to reproduce the bug?

```css
margin: ${({ isSomeProperty }) =>
        isSomeProperty === true
          ? `${someVariable} 16px`
          : `${someOtherVariable} 16px`};
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "unit-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

`13.3.2`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

"CLI with `stylelint "**/*.js" --config ../config/stylelintrc.js`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Styled Components properties.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
 61:35  ✖  Unexpected unknown unit "px`"   unit-no-unknown
```

**Comment by jeddy3** [comment]
@plumbblake Thanks for the report and for using the template.

I'm going to close this as a duplicate of https://github.com/stylelint/stylelint/issues/4728.

That issue is marked a ready to implement. Please consider [contributing](https://stylelint.io/contributing) a fix if you have time.
